### PR TITLE
Api Acess Page

### PR DESCRIPTION
**Pull request by murilovmachado**
This PR implements the API Access Page, that consists of two main sections:
- API Endpoints Section (Finished)
- Kubeconfig Section (to be finished in another PR)

Screen Shot 2019-11-05 at 13 23 11

### CODE REVIEW

File: src/app/plugins/kubernetes/components/apiAccess/endpoints/EndpointsListPage.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  return (
+    <>
+      <h2>API Endpoints</h2>
+      <ListPage />
+      <a className={classes.link} href="/clarity/docs/qbert/">See qbert API Documentation</a>
+    </>
+  )
+}
+
+const getColumns = () => [
```
Comment (brennancheung) [inline src/app/plugins/kubernetes/components/apiAccess/endpoints/EndpointsListPage.js]: Doesn't need to be a function.

File: src/app/plugins/kubernetes/components/apiAccess/kubeConfig/actions.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import createCRUDActions from 'core/helpers/createCRUDActions'
+
+export const kubeConfigCacheKey = 'apiAccess-kubeConfig'
+
+const kubeConfigActions = createCRUDActions(kubeConfigCacheKey, {
+  // TODO: implement list fetching real data
+  listFn: async (params, loadFromContext) => {
+    const kubeConfig = Promise.resolve([
+      { cluster: 'klee-test', url: 'klee-test-5ab3b7a0-api.df-us-mpt1-kvm.platform9.net' },
```
Comment (brennancheung) [inline src/app/plugins/kubernetes/components/apiAccess/kubeConfig/actions.js]: Let's keep actual server FQDN's out of our repo for security reasons.

File: src/app/core/components/listTable/ListTableHead.js [hunk 3]
```diff
@@ -74,10 +74,10 @@ class ListTableHead extends React.PureComponent {
       showCheckboxes,
     } = this.props
 
-    const headerCheckbox = !blankFirstColumn
+    const headerCheckbox = !blankFirstColumn && showCheckboxes
```
Comment (brennancheung) [inline src/app/core/components/listTable/ListTableHead.js]: Can you clean up this ternary?  The `?` and `:` are too far apart.

File: src/app/plugins/kubernetes/components/apiAccess/kubeConfig/actions.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import createCRUDActions from 'core/helpers/createCRUDActions'
+
+export const kubeConfigCacheKey = 'apiAccess-kubeConfig'
+
+const kubeConfigActions = createCRUDActions(kubeConfigCacheKey, {
+  // TODO: implement list fetching real data
+  listFn: async (params, loadFromContext) => {
+    const kubeConfig = Promise.resolve([
```
Comment (xantorres) [inline src/app/plugins/kubernetes/components/apiAccess/kubeConfig/actions.js]: `Promise.resolve' is useless here, you can just return the array as is

**Pull request merged**